dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -211,7 +211,7 @@ internal static List<RestApiOperation> CreateRestApiOperations(OpenApiDocument d
                     path: path,
                     method: new HttpMethod(method),
                     description: string.IsNullOrEmpty(operationItem.Description) ? operationItem.Summary : operationItem.Description,
-                    parameters: CreateRestApiOperationParameters(operationItem.OperationId, operationItem.Parameters),
+                    parameters: CreateRestApiOperationParameters(operationItem.OperationId, pathItem.Parameters.Union(operationItem.Parameters).ToList()),
```

Comment:
nit: consider changing the `parameters` parameter type of the `CreateRestApiOperationParameters` method from IList<T> to IEnumerable<T> as an alternative to the `ToList()` call.

Reply:
fixed